CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.20)
 
 
 set(ZITI_SDK_DIR "" CACHE FILEPATH "developer option: use local ziti-sdk-c checkout")
-set(ZITI_SDK_VERSION "0.32.8" CACHE STRING "ziti-sdk-c version or branch to use")
+set(ZITI_SDK_VERSION "main" CACHE STRING "ziti-sdk-c version or branch to use")
```

Comment:
IMO we should stick to versioned releases.

Reply:
this is to make a build for test. Not merging like this